thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/detector/email/filter/AlertFilterUtil.java

```diff
@@ -6,22 +6,38 @@
 import java.util.List;
 
 
+/**
+ * Utility class to evaluate alert filter
+ */
 public class AlertFilterUtil {
   public static final Integer PRECISION_INDEX = 0;
   public static final Integer RECALL_INDEX = 1;
+  public static final Integer TOTAL_ANOMALIES_INDEX = 0;
+  public static final Integer TOTAL_RESPONSES_INDEX = 1;
+  public static final Integer TRUE_ANOMALY_INDEX = 2;
+  public static final Integer FALSE_ALARM_INDEX = 3;
+  public static final Integer NONACTIONALBLE_INDEX = 4;
 
   private static final Integer NUM_EVALUATE_METRICS = 2;
+  private static final Integer NUM_EVALUTATE_FEEDBACK_METRICS = 5;
+
   private AlertFilter alertFilter;
 
   public AlertFilterUtil(AlertFilter alertFilter){
     this.alertFilter = alertFilter;
   }
 
-  public double[] getEvalResults(List<MergedAnomalyResultDTO> mergedAnomalyResultDTOS) throws Exception {
+  /**
+   * Evaluate alert filter given merged anomalies and output precision and recall
+   * @param mergedAnomalyResultDTOS
+   * @return Array of precision and recall
+   * @throws Exception
+   */
+  public Double[] getEvalResults(List<MergedAnomalyResultDTO> mergedAnomalyResultDTOS) throws Exception {
     int TP = 0;
     int FP = 0;
     int FN = 0;
-    double[] evals = new double[NUM_EVALUATE_METRICS];
+    Double[] evals = new Double[NUM_EVALUATE_METRICS];
```

Comment:
Would it make sense to have an object "AlertFilterEvauationResult" instead of a double array, and returning that as result. 
This way you would not have to maintain the index names and use it outside this util class.